java/README.md

```diff
@@ -1,16 +1,24 @@
-You will need maven to compile and run this example.
-On OS X, you can install maven with Homebrew by running `brew install maven`.
+Use SigOpt's [Java API Client](https://sigopt.com/docs/overview/java) to tune the [Franke function](http://www.sfu.ca/~ssurjano/franke2d.html)
+
+# Installation
+You will need [maven](https://maven.apache.org/) to compile and run this example.
+On OS X, you can install maven with Homebrew by running 
+
+```bash
+brew install maven
+```
+
+Then, install SigOpt's [Java API Client](https://sigopt.com/docs/overview/java). Follow the instructions on this page for accessing your SigOpt API Token. You do not need to follow the step for the `pom.xml` file, since it is already in this repository.
```

Comment:
Oh I am not sure, I just saw the step about project settings and noticed that it didn't seem to be included here